src/fund/components/FundCardCurrencyLabel.tsx

```diff
@@ -0,0 +1,26 @@
+import { forwardRef } from 'react';
+import { useTheme } from '../../core-react/internal/hooks/useTheme';
+import { cn, text } from '../../styles/theme';
+import type { FundCardCurrencyLabelPropsReact } from '../types';
+
+export const FundCardCurrencyLabel = forwardRef<
+  HTMLSpanElement,
+  FundCardCurrencyLabelPropsReact
+>(({ currencySign }, ref) => {
+  const componentTheme = useTheme();
```

Comment:
Same as above but you should only need to pass this componentTheme down in the top level component